pkg/visibility/api/rest/pending_workloads_cq_test.go

```diff
@@ -298,15 +316,17 @@ func TestPendingWorkloadsInCQ(t *testing.T) {
 				manager.AddOrUpdateWorkload(w)
 			}
 
-			for _, cq := range tc.clusterQueues {
-				info, err := pendingWorkloadsInCqRest.Get(ctx, cq.Name, tc.queryParams)
+			for req, resp := range tc.wantResponse {
+				info, err := pendingWorkloadsInCqRest.Get(ctx, req.queueName, req.queryParams)
 				if err != nil {
```

Comment:
We should always check the errors. For example, if `err` is nil, and `resp.wantErr` is nil, the test, as written, wouldn't report that the error is not found.